codespell_lib/_codespell.py

```diff
@@ -273,6 +273,13 @@ def parse_options(args):
                         'to include (when "-D -" or no "-D" is passed). '
                         'Current options are:' + builtin_opts + '\n'
                         'The default is %(default)r.')
+    parser.add_argument('--ignore-regex',
+                        action='store', type=str,
+                        help='regular expression which is used to find '
+                             'patterns to ignore by treating as whitespace. '
+                             'When writing regexes, consider ensuring there '
+                             'is boundary non-word chars, e.g., '
```

Comment:
Nit sorry, grammar wise it should be "ensuring there are boundary..."

Reply:
Done